example/notify_expire_entry.c

```diff
@@ -0,0 +1,350 @@
+/*
+  FUSE: Filesystem in Userspace
+  Copyright (C) 2016 Nikolaus Rath <Nikolaus@rath.org>
+
+  This program can be distributed under the terms of the GNU GPLv2.
+  See the file COPYING.
+*/
+
+/** @file
+ *
+ * This example implements a file system with a single file whose
+ * file name changes dynamically to reflect the current time.
```

Comment:
This looks very close to the existing notify_inval_entry.c example. Would it be possible to instead modify the existing example to add a --expire flag (to use the different notification function) to reduce code duplication?